<ui version="4.0" >
 <class>ModelProperties</class>
 <widget class="QDialog" name="ModelProperties" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Model Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>522</width>
        <height>515</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Model properties</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>&lt;h1>Model properties&lt;/h1></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>&lt;b>Name&lt;/b></string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>&lt;b>Description&lt;/b></string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="name" />
         </item>
         <item row="2" column="1" >
          <widget class="QTextEdit" name="description" >
           <property name="tabChangesFocus" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>&lt;b>Features&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-f57a8f871fc97ceb2f6daa43528fd640ee63b4f4">?&lt;/a>&lt;/b></string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="openExternalLinks" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>&lt;b>Tags&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-09efbe5fd6809ae0d90543adf92014b8eb9ef1bf">?&lt;/a>&lt;/b></string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="openExternalLinks" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="tags" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>&lt;b>Card spacing&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-59a81e35b6afb23930005e943068945214d194b3">?&lt;/a>&lt;/b></string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="openExternalLinks" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_16" >
             <property name="text" >
              <string>Minimum spacing&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-4ee3a58e2fdd61da8ef81984213862d3bc0ed4bd">?&lt;/a></string>
             </property>
             <property name="openExternalLinks" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLineEdit" name="initialSpacing" />
           </item>
           <item row="0" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_24" >
             <property name="text" >
              <string>Minimum interval multiplier&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-51074b785bb6049b44fce9f18fca198100c4d2f7">?&lt;/a></string>
             </property>
             <property name="openExternalLinks" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QLineEdit" name="spacing" />
           </item>
          </layout>
         </item>
         <item row="4" column="1" >
          <widget class="QLineEdit" name="decorators" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>522</width>
        <height>515</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Fields</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>&lt;h1>Fields&lt;/h1></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10" >
         <property name="text" >
          <string>A flashcard is made from a number of fields, like "meaning", "notes", etc.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="fieldList" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="fieldAdd" >
             <property name="text" >
              <string>&amp;Add</string>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fieldDelete" >
             <property name="text" >
              <string>&amp;Delete</string>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fieldUp" >
             <property name="toolTip" >
              <string>Move selected field up</string>
             </property>
             <property name="text" >
              <string>Move &amp;Up</string>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fieldDown" >
             <property name="toolTip" >
              <string>Move selected field down</string>
             </property>
             <property name="text" >
              <string>Move Dow&amp;n</string>
             </property>
             <property name="autoDefault" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="fieldEditBox" >
         <property name="title" >
          <string>Field properties</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_18" >
              <property name="text" >
               <string>&lt;b>Unique?</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="fieldName" />
            </item>
            <item row="2" column="1" >
             <widget class="QCheckBox" name="fieldUnique" >
              <property name="text" >
               <string>Prevent me from entering the same thing in this field twice</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_20" >
              <property name="text" >
               <string>&lt;b>Name&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_19" >
              <property name="text" >
               <string>&lt;b>Description&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_17" >
              <property name="text" >
               <string>&lt;b>Required?&lt;/b></string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QCheckBox" name="fieldRequired" >
              <property name="text" >
               <string>Prevent new cards from being added if this field is blank</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" >
             <widget class="QLabel" name="label_22" >
              <property name="text" >
               <string>&lt;b>Features&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-f57a8f871fc97ceb2f6daa43528fd640ee63b4f4">?&lt;/a>&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="openExternalLinks" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QTextEdit" name="fieldDescription" >
              <property name="tabChangesFocus" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1" >
             <layout class="QVBoxLayout" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <property name="margin" >
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="fieldFeatures" />
              </item>
             </layout>
            </item>
            <item row="4" column="0" >
             <widget class="QLabel" name="label_21" >
              <property name="text" >
               <string>&lt;b>Numeric?&lt;/b></string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="QCheckBox" name="numeric" >
              <property name="text" >
               <string>Sort this field using numeric order instead of string order</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>522</width>
        <height>515</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Cards</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&lt;h1>Card models&lt;/h1></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11" >
         <property name="text" >
          <string>One or more cards are generated for each piece of information you enter into Anki. Here you can control how many cards are generated, and what they look like. Spacing is the amount of time before showing a different card for the same piece of information.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="cardList" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="cardAdd" >
             <property name="text" >
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cardToggle" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cardDelete" >
             <property name="text" >
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cardUp" >
             <property name="toolTip" >
              <string>Move selected card model up</string>
             </property>
             <property name="text" >
              <string>Move &amp;Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cardDown" >
             <property name="toolTip" >
              <string>Move selected card model down</string>
             </property>
             <property name="text" >
              <string>Move Dow&amp;n</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="cardEditBox" >
         <property name="title" >
          <string>Edit card</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="1" >
             <widget class="QTextEdit" name="cardDescription" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="horizontalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="tabChangesFocus" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_13" >
              <property name="text" >
               <string>&lt;b>Description&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0" >
             <widget class="QLabel" name="label_23" >
              <property name="text" >
               <string>&lt;b>Question hiding&lt;/b></string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" >
             <widget class="QCheckBox" name="questionInAnswer" >
              <property name="text" >
               <string>Hide the question when showing answer</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QTextEdit" name="cardAnswer" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="horizontalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="tabChangesFocus" >
               <bool>true</bool>
              </property>
              <property name="acceptRichText" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_15" >
              <property name="text" >
               <string>&lt;b>Answer format&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-b8916ff117aa0da4a414ce9b9b9be4a232eab2f4">?&lt;/a>&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="openExternalLinks" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="cardName" />
            </item>
            <item row="2" column="1" >
             <widget class="QTextEdit" name="cardQuestion" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>60</height>
               </size>
              </property>
              <property name="horizontalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="tabChangesFocus" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_12" >
              <property name="text" >
               <string>&lt;b>Name/tag&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-09efbe5fd6809ae0d90543adf92014b8eb9ef1bf">?&lt;/a>&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="openExternalLinks" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_14" >
              <property name="text" >
               <string>&lt;b>Question format&lt;a href="http://ichi2.net/anki/wiki/Key_Terms_and_Concepts#head-b8916ff117aa0da4a414ce9b9b9be4a232eab2f4">?&lt;/a>&lt;/b></string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="openExternalLinks" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>name</tabstop>
  <tabstop>tags</tabstop>
  <tabstop>description</tabstop>
  <tabstop>initialSpacing</tabstop>
  <tabstop>spacing</tabstop>
  <tabstop>decorators</tabstop>
  <tabstop>fieldList</tabstop>
  <tabstop>fieldAdd</tabstop>
  <tabstop>fieldDelete</tabstop>
  <tabstop>fieldUp</tabstop>
  <tabstop>fieldDown</tabstop>
  <tabstop>fieldName</tabstop>
  <tabstop>fieldDescription</tabstop>
  <tabstop>fieldUnique</tabstop>
  <tabstop>fieldRequired</tabstop>
  <tabstop>numeric</tabstop>
  <tabstop>fieldFeatures</tabstop>
  <tabstop>cardList</tabstop>
  <tabstop>cardAdd</tabstop>
  <tabstop>cardToggle</tabstop>
  <tabstop>cardDelete</tabstop>
  <tabstop>cardUp</tabstop>
  <tabstop>cardDown</tabstop>
  <tabstop>cardName</tabstop>
  <tabstop>cardDescription</tabstop>
  <tabstop>cardQuestion</tabstop>
  <tabstop>cardAnswer</tabstop>
  <tabstop>questionInAnswer</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModelProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModelProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
